Geometry/__init__.py

```diff
@@ -0,0 +1,249 @@
+import cmath
```

Comment:
Hmm... I don't think we want to add another module at the root of the repository. Additionally, without any module-level documentation, I don't wish to add it to the repo. Thus, here are two suggestions:

- [ ] Make a new sub-package under `electricpy` as: `electricpy/geometry/...files...` (note the capitalization is important, it's more "pythonic" as a lowercase name)
- [ ] Add a module-level docstring (using the tripple-quotes, i.e., `"""My Comment"""`) to explain the purpose of the file and how it's used in the broader scope of the `electricpy` package.

Reply:
done !!!!!